### PR TITLE
Nathan small UI changes

### PR DESCRIPTION
**Pull request by nepoche**
These PR has changes to implement the following issues:
- After copying note, text on AccountsManager, withdraw modal, does not display properly
- Change name of second "Note" to "Recipient"
- Remove the entire greeting
- Some fonts in dark mode are not properly colored
- Fix ZkProff, change to zkBridge

### CODE REVIEW

File: packages/mixer/src/components/Withdraw/WithdrawingModal.tsx
Comment (nepoche) [inline packages/mixer/src/components/Withdraw/WithdrawingModal.tsx:73]: Had some trouble trying to style this component with the theme selection

**Review by drewstone — APPROVED** [review]
Will wait for @AhmedKorim to approve before merging.

**Comment by drewstone** [comment]
Actually since this is merging in `ahmed.contracts` we can handle it now.

**Pull request merged**
